conf/defaults.ini

```diff
@@ -1168,6 +1168,11 @@ global_correlations = -1
 # This is not strictly enforced yet, but will be enforced over time.
 alerting_rule_group_rules = 100
 
+# Limit the number of query evaluation results per alert rule.
+# If the condition query of an alert rule produces more results than this limit,
+# the evaluation results in an error.
+alerting_rule_evaluation_results = -1
```

Comment:
I added it here because `alerting_rule_group_rules` is also here, but maybe `unified_alerting` is a better place for this, not sure.

Reply:
The logic for the other (incomplete) setting was - even though this is not implemented by the grafana quota service, the user doesn't necessarily care about that implementation detail, and it might be nice to have all limits together :shrug: